documentation/release-process.md

```diff
@@ -3,16 +3,14 @@
 
 # Release Process
 
-## Merge to Release Branch
+## Prepare the release branch
 
 1. Merge from the `main` branch to the appropriate release branch (e.g. `release/5.0`). Note that for patch releases, fixes should be made directly to the appropriate release branch and we do not merge from the `main` branch. Note that it is acceptable to use a release/major.x branch. Alternatively, you can create a new release branch for the minor version. See [additional branch steps](#additional-steps-when-creating-a-new-release-branch) below.
-1. In `/eng/Versions.props`, update `dotnet/diagnostics` dependencies to versions from the corresponding release of the `dotnet/diagnostics` repo. Note this should be done using darc. See [updating dependencies](#updating-dependencies).
-1. In `/eng/Versions.props`, ensure that `<BlobGroupBuildQuality>` is set appropriately. See the documentation next to this setting for the appropriate values. In release branches, its value should be `release`. This setting, in combination with the version settings, determine for which 'channel' the aks.ms links are created.
-1. When preparing for a GA or servicing release, you will need to update `PreReleaseVersionLabel` to `rtm` (for the GA release) or `servicing` (for subsequent patch releases), the `DotnetFinalVersionKind` to 'release' and remove the `PreReleaseVersionIteration`. See https://github.com/dotnet/dotnet-monitor/pull/1970/files for an example.
-
+1. Review and merge in any outstanding dependabot PRs for the release branch.
+1. Run the [Update release version](https://github.com/dotnet/dotnet-monitor/actions/workflows/update-release-version.yml) workflow, setting `Use workflow from` to the release branch and correctly setting the `Release type` and `Release version` options. (*NOTE:* Release version should include only major.minor.patch, without any extra labels). Review and merge in the PR created by this workflow.
+1. If you merged from `main` in step 1, repeat the above step for the `main` branch, using `alpha` as the `Release type`.
```

Comment:
`alpha` is only used when jumping from one major version to the next. Otherwise, it is typically `preview` that is used in the `main` branch.